.github/workflows/postman.yml

```diff
@@ -0,0 +1,70 @@
+          sed -i 's/0d00010776bf65acf2d25e9dcf4c716f5f39f201dccbfa173ad9c7f1c1dbcb8a86776b4d0000000101002100000000000000000000000000000000000000000000000000012901c1cf3900000200450600040279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798010000000000000000000000000000000000000000000000017afba303493b7ff8000800000200450600040279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798010000000000000000000000000000000000000000000000017afba303493b7f9402004506000402c6047f9441ed7d6d3045406e95c07cd85c778e4b8cef3ca7abac09b95c709ee5010000000000000000000000000000000000000000000000000000000000000064000c003430303030366336663631363432303734363537333734363936653637323037343732363136653733363136333734363936663665/"{{unsigned_transaction}}"/g' gateway-api-spec.yaml &&\
+          sed -i 's/0279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798/"{{public_key_hex}}"/g' gateway-api-spec.yaml
+      - name: Update Postman's Gateway API collection from the spec
+        run: |
+          npx @apideck/portman -l gateway-api-spec.yaml --syncPostman true -o tmp.collection.json --envFile .env --cliOptionsFile cliopts.json \
+            --postmanUid 14449947-b6b059b1-67db-46ea-901b-cb446c3c3a1e
+      - name: Download the collection
+        run: |
+          curl -X GET -H "X-API-KEY:${{secrets.POSTMAN_API_TOKEN}}" https://api.getpostman.com/collections/14449947-b6b059b1-67db-46ea-901b-cb446c3c3a1e > tmp.collection.json
+      - name: Add a timestamp to the name
+        run: |
+          sed -i 's/"name":".*","description":"This API/"name":"Gateway API (develop) ${{steps.date.outputs.date}} UTC","description":"This API/' tmp.collection.json
+      - name: Change the baseUrl variable name
+        run: |
+          sed -i 's/{{baseUrl}}/{{gatewayBaseUrl}}/g' tmp.collection.json
+      - name: Update the collection
+        run: |
+          curl -X PUT -H "X-API-KEY:${{secrets.POSTMAN_API_TOKEN}}" -H "Content-Type: application/json" \
+            https://api.getpostman.com/collections/14449947-b6b059b1-67db-46ea-901b-cb446c3c3a1e --data "@tmp.collection.json"
+      - name: Validate the collection against RCnet
+        run: |
+          newman run tmp.collection.json \
+            --environment https://api.getpostman.com/environments/3671706-92569d6f-4d81-4b15-a5a2-d65afc8fdc26?apikey=${{secrets.POSTMAN_API_TOKEN}} \
+            --folder Status --folder Account --folder Token --folder Validator --bail
```

Comment:
What does the --bail parameter do out of interest?

Reply:
`--bail` tells Newman to fail fast, like it will immediately stop the test runner if a single test fails. 

Actually I don't think it makes sense to bail here. I'll remove it